build.gradle

```diff
@@ -51,6 +51,8 @@ configurations {
 
     geolite2 { canBeConsumed = false }
 
+    mockitoAgent { canBeConsumed = false }
```

Comment:
See https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#0.3 for java 21 compat.

Reply:
In my testing (see java version below), the warnings I was seeing have all been resolved with the changes in `build.gradle`.  

```
openjdk version "21.0.5" 2024-10-15
OpenJDK Runtime Environment Homebrew (build 21.0.5)
OpenJDK 64-Bit Server VM Homebrew (build 21.0.5, mixed mode, sharing)
```

Reply:
Oh cool, that's less invasive than I was envisioning after reading that article

Reply:
And now I read the rest of the `build.gradle` I see the invasive part 🤦‍♂️